README.md

```diff
@@ -81,15 +81,15 @@ The camelCase D functions will be PascalCase in Excel.
 
 ## Python versions
 
-Since autowrap depends on PyD, the python version must be explicitly stated
-as a dub configuration and defaults to 3.6. To use another version, pass
-`-c $CONFIG` to dub where `$CONFIG` is one of:
-
-* python27
-* python34
-* python35
-* python36
-
+Since autowrap depends on PyD, the version of python to use must be specified.
+The default is `python37`. You can either specify the configuration explicity,
```

Comment:
env should then be the default and shouldn't involve the user doing anything. It should just work.

Reply:
Fine by me, but that's a breaking change that might be annoying for people to work around.

Reply:
semver will take care of that.

Reply:
done

Reply:
Well, I made it the default, it still requires the environment variables being set before the build, but I really don't know any way around that.